### PR TITLE
Added Khmer language support for reports; COUNTRY=cambodia

### PR DESCRIPTION
**Pull request by HarryMytilinaios**
Adds support for Khmer language in reports generation for cambodia.

### CODE REVIEW

File: frontend/src/components/MapView/LeftPanel/AnalysisPanel/AnalysisTable/ExposureAnalysisTable/index.tsx [hunk 1]
```diff
@@ -197,7 +197,7 @@ const ExposureAnalysisTable = memo(
           rowsPerPage={rowsPerPage}
           page={page}
           onPageChange={handleChangePage}
-          onChangeRowsPerPage={handleChangeRowsPerPage}
+          onRowsPerPageChange={handleChangeRowsPerPage}
```
Comment (ericboucher) [inline frontend/src/components/MapView/LeftPanel/AnalysisPanel/AnalysisTable/ExposureAnalysisTable/index.tsx:200]: why the name change?

File: frontend/src/components/Common/ReportDialog/reportDoc.tsx [hunk 2]
```diff
@@ -11,20 +12,33 @@ import { Theme } from '@material-ui/core';
 import { TableRow as AnalysisTableRow } from 'context/analysisResultStateSlice';
 import { getLegendItemLabel } from 'components/MapView/utils';
 import { LegendDefinition, ReportType } from 'config/types';
-import { TFunction } from 'utils/data-utils';
 import { Column } from 'utils/analysis-utils';
+import RobotoFont from 'fonts/Roboto-Medium.ttf';
```
Comment (ericboucher) [inline frontend/src/components/Common/ReportDialog/reportDoc.tsx:16]: Looks like Roboto-Regular.tff is added but never used?

File: frontend/src/components/Common/ReportDialog/reportDoc.tsx [hunk 3]
```diff
@@ -11,20 +12,33 @@ import { Theme } from '@material-ui/core';
...
+
+const makeStyles = (theme: Theme, selectedLanguage: string) =>
   StyleSheet.create({
     page: {
       flexDirection: 'column',
       backgroundColor: '#FFF',
       paddingBottom: 25,
       paddingTop: '1vh',
-      fontFamily: 'Roboto',
+      fontFamily: selectedLanguage === 'kh' ? 'Khmer' : 'Roboto',
```
Comment (ericboucher) [inline frontend/src/components/Common/ReportDialog/reportDoc.tsx:41]: Let's create a small utility function for mapping. Either as a map or a switch maybe?

File: frontend/src/components/MapView/LeftPanel/AnalysisPanel/AnalysisTable/ExposureAnalysisTable/index.tsx (hunk 1 above)
Comment (HarryMytilinaios) [reply frontend/src/components/MapView/LeftPanel/AnalysisPanel/AnalysisTable/ExposureAnalysisTable/index.tsx:200]: That was a warning from the upgrade of the Materia UI the `onChangeRowsPerPage` was deprecated in favor of `onRowsPerPageChange`  so I changed it to stop the console errors from Material

File: frontend/src/components/Common/ReportDialog/reportDoc.tsx (hunk 2 above)
Comment (HarryMytilinaios) [reply frontend/src/components/Common/ReportDialog/reportDoc.tsx:16]: Robot is used conditionally below

File: frontend/src/components/Common/ReportDialog/reportDoc.tsx (hunk 3 above)
Comment (HarryMytilinaios) [reply frontend/src/components/Common/ReportDialog/reportDoc.tsx:41]: on it

Comment (HarryMytilinaios) [reply frontend/src/components/Common/ReportDialog/reportDoc.tsx]: Done

File: frontend/src/components/Common/ReportDialog/reportDoc.tsx (hunk 2 above)
Comment (ericboucher) [reply frontend/src/components/Common/ReportDialog/reportDoc.tsx:16]: yes but only the medium one, no? not the regular file that you also committed to the repo

**Comment by wadhwamatic** [comment]
This is working well, thanks @HarryMytilinaios! I'll work with the Cambodia office to fill in some missing translations which we can add separately.

File: frontend/src/components/Common/ReportDialog/reportDoc.tsx (hunk 2 above)
Comment (HarryMytilinaios) [reply frontend/src/components/Common/ReportDialog/reportDoc.tsx:16]: Yes you are right I wanted actually to add the Regular so the Medium is not necessary sorry for the misconception, those were a part of some testing I'll fix it

**Review by ericboucher — APPROVED** [review]

**Pull request merged**